src/test/java/useCases/impactor/PropertyImpactorTest.java

```diff
@@ -0,0 +1,58 @@
+package useCases.impactor;
+
+import entity.Player;
+import entity.Property;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class PropertyImpactorTest {
+
+    private static Property property;
+    private static Player player1;
+    private static Player player2;
+
+    /**
+     * setup method for this test
+     */
+    @BeforeAll
+    public static void setUp() {
+        property = new Property(1,
+                "1",
+                new ArrayList<>(Arrays.asList(1, 2, 3, 0)),
+                new ArrayList<>(Arrays.asList(0, 1, 2, 3)));
+        player1 = new Player(1,1);
```

Comment:
Small convention issue, shoud be: 
```suggestion
        player1 = new Player(1, 1);
```